requirements.txt

```diff
@@ -1,6 +1,5 @@
-black == 19.10.b0
-pydocstyle == 4.0.1
-pytest == 5.2.2
-pytest-cov == 2.8.1
-pytest-flake8 == 1.0.4
-
+pytest-cov~=2.8.1
```

Comment:
Could you add `pytest` back here? I don't like relying on transitive dependencies for tools the project actually depends on.